share/pvss/pvss.go

```diff
@@ -15,13 +15,28 @@ package pvss
 
 import (
 	"errors"
+	"hash"
 
 	"github.com/dedis/crypto"
 	"github.com/dedis/crypto/proof"
-	"github.com/dedis/crypto/util/random"
 	"github.com/dedis/crypto/share"
+	"github.com/dedis/crypto/util/random"
 )
 
+// NOTE. Here, we need need the full suite definition for using the proof/
+// package while we only use Hash.
```

Comment:
Exactly, proof/ needs to be slimmed down more and packaged into sub-packages. Unfortunatly, this problem is likely to be occuring in many places in our code, rendering the approach quite expensive compared to the advantages in my opinion.